### DISCUSSION TITLE
캐릭터 컨트롤에 관한 질문입니다.

### DISCUSSION THREAD

**Discussion by FOJF** [naverz/zepeto-studio-kor#271] [5. Q&A]
캐릭터를 연속으로 점프하게 하는 것은 불가능한가요??

2단 점프나 그 이상 가능하다면 방법이 궁금합니다.

**Comment by x4cf** [comment]
가능합니다.

구현은 방법이 너무 다양하며 원하는 조작감과 필요에 따라 직접 제작하시는게 좋습니다.
